### DISCUSSION TITLE
R package... very common demographic method

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/analytics-corner#14] [Ideas]
re @usamabilal package idea 

> btw i have an idea for an R package... very common demographic method that has no package at all to speak of
> Bricia had to get stuff rom here and there to implement it, and i'm having to do the same thing now maybe for the future :stuck_out_tongue:

(creating this cause ideas are valuable)

**Comment by usamabilal** [comment]
To be precise, I'm talking about the Arriaga decomposition:
Paper here https://u.demog.berkeley.edu/~jrw/Biblio/Eprints/%20A-C/arriaga.1984.pdf
one example of implementation here: https://github.com/Hong-Kong-Longevity/Longevity/blob/main/Arriaga%2095CI.R
Another (in stata) here https://osf.io/e39y2

I'll be implementing this myself for another project and once i have that i'll think about how to "package" this.
